requirements.txt

```diff
@@ -0,0 +1,7 @@
+aiofiles
```

Comment:
Ideally this should be inferred from the stuff defined in `setup.py` but I'm not well versed enough in python to understand the difference 🤔